src/init.ml

```diff
@@ -0,0 +1,196 @@
+open! Stdune
+open! Import
+
+module Project = struct
```

Comment:
Do we really need this extra `Project` wrapper? It doesn't seem like `Project.Kind.t` is any clearer than `Kind.t`. If anything it's a `Stanza.Kind.t`, but in which case `Stanza_kind.t` is just as good.`